packages/apps/graph/src/scripts/devnet/simulate.ts

```diff
@@ -172,6 +181,21 @@ export async function simulate({
         type: transferType,
       });
 
+      // Increase total transfers
+      totalTransfers++;
+
+      // Check if max transfers has been reached
+      if (totalTransfers >= maxTransfers) {
+        logger.info('Maximum number of transfers reached');
+        break;
+      }
+
+      // Check if duration has been reached
+      if (Date.now() - startTime >= duration) {
+        logger.info('Duration reached');
+        break;
+      }
```

Comment:
The reason behind separating it into 2 `if` statements it just so we can have conclusive logging